lib/rx/configuration.rb

```diff
@@ -40,6 +40,7 @@ def connection
         # conn.request(:curl, ::Logger.new(STDOUT), :warn) unless Rails.env.production?
         # conn.response(:logger, ::Logger.new(STDOUT), bodies: true) unless Rails.env.production?
 
+        conn.response :betamocks if Settings.mhv.rx.mock
```

Comment:
@omgitsbillryan I see a variety of strategies for this conditional. For example, some folks call a private method defined elsewhere. In all cases I've seen they're ultimately just checking the settings flag. Any preference here?

Reply:
Eh, a lot of the `configuration` classes inherit their connection, in which case, it makes sense for the child class to implement:
```ruby
def mock_enabled?
  Settings.mhv.rx.mock || false
end
```
In this case, you're defining your own `conneciton`, so what you have is fine IMO.